containers/redeem/RedeemCouponSection.js

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useCallback } from 'react';
```

Comment:
Move this file to `protonvpn-settings` project please. This is a specific logic.